README.md

```diff
@@ -457,6 +472,149 @@ Uploads interrupted either due to bad internet connection or manual interruption
 - No progress bars for resumable uploads as it messes up with output.
 - You can interrupt many times you want, it will resume ( hopefully ).
 
+## Additional Usage
+
+### Synchronisation
+
+This repo also provides an additional script ( [sync.sh](https://github.com/labbots/google-drive-upload/blob/master/sync.sh) ) to utilise upload.sh for synchronisation jobs, i.e background jobs.
+
+#### Basic Usage
+
+To create a sync job, just run
+
+`gsync folder_name -d gdrive_folder`
+
+Here, folder_name is the local folder you want to sync and gdrive_folder is google drive folder name.
+
+In the local folder, all the contents present or added in the future will be automatically uploaded.
+
+Note: Giving gdrive_folder is optional, if you don't specify a name with -d/--directory flags, then it will take the name of the local folder.
+
+Also, gdrive folder creation works in the same way as gupload command.
+
+Default wait time: 3 secs ( amount of time to wait before checking new files ).
+
+Default gupload arguments: None ( see -a/--arguments section below ).
+
+<a id="custom-flags-sync"></a>
+#### Custom Flags
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [[no-duplicate-headings] Do not use headings with similar content (299:1)](https://app.codacy.com/manual/labbots/google-drive-upload/pullRequest?prid=5619177)